src/Components/Blazor/DevServer/src/Microsoft.AspNetCore.Blazor.DevServer.csproj

```diff
@@ -11,15 +11,14 @@
     <Description>Development server for use when building Blazor applications.</Description>
     <!-- Set this to false because assemblies should not reference this assembly directly, (except for tests, of course). -->
     <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
+
+    <!-- This is so that we add the FrameworkReference to Microsoft.AspNetCore.App -->
+    <UseLatestAspNetCoreReference>true</UseLatestAspNetCoreReference>
   </PropertyGroup>
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Blazor.Server" />
-    <Reference Include="Microsoft.AspNetCore.Components.Server" />
-    <Reference Include="Microsoft.AspNetCore.ResponseCompression" />
-    <Reference Include="Microsoft.AspNetCore" />
     <Reference Include="Microsoft.Extensions.CommandLineUtils.Sources" />
-    <Reference Include="Microsoft.Extensions.Hosting" />
```

Comment:
We'll have to add all these back in `master`, right? Or do you plan to stop this whole change from flowing to `master` somehow?

Reply:
We will likely have to resolve/revert the conflict at one point and that should be it.